pkg/util/binding.go

```diff
@@ -72,28 +72,28 @@ func ConvertToClusterNames(clusters []workv1alpha2.TargetCluster) sets.Set[strin
 }
 
 // MergeTargetClusters will merge the replicas in two TargetCluster
-func MergeTargetClusters(old, new []workv1alpha2.TargetCluster) []workv1alpha2.TargetCluster {
+func MergeTargetClusters(old_cluster, new_cluster []workv1alpha2.TargetCluster) []workv1alpha2.TargetCluster {
```

Comment:
Hi~ @Affan-7 Is it better to use hump case for parameters? Like `oldCluster`.

Reply:
Thanks for the suggestion, I have edited the code.